yaml/xyz/openbmc_project/State/BMC/Redundancy.interface.yaml

```diff
@@ -19,6 +19,21 @@ properties:
           If redundancy is currently enabled.  In general, this means that the
           BMCs are configured as active and passive and that the passive is able
           to be failed over to.
+    - name: DisableRedundancyOverride
```

Comment:
I hope implementation will take care to disable sync while setting this property, right? Or let user disable sync directly from the data sync?